pkg/server/server.go

```diff
@@ -42,7 +41,7 @@ import (
 	"k8s.io/kubernetes/pkg/genericcontrolplane"
 
 	configroot "github.com/kcp-dev/kcp/config/root"
-	"github.com/kcp-dev/kcp/config/system-crds"
+	systemcrds "github.com/kcp-dev/kcp/config/system-crds"
```

Comment:
Is this a no-op?

Reply:
Yeh, Goland seemed to think it was a good idea 🤷